# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: mozglue/tests/gtest/moz.build
# Commit: 9b55c0eea2be
# Full Hash: 9b55c0eea2befe5abf98dfa0b168d4ac9cc78c8d
# Author: Paul Adenot <paul@paul.cx>
# Date: 2024-10-21 16:14:35
# Regressor Bug: 1717448
# File Overlap Count: 1
# Description:
#   Bug 1717448 - Add some tests for Gecko integration. r=glandium
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D217299
# ==============================================================================

diff -r ffa33e073c36 -r 9b55c0eea2be mozglue/tests/gtest/moz.build
--- a/mozglue/tests/gtest/moz.build	Mon Oct 21 12:54:56 2024 +0000
+++ b/mozglue/tests/gtest/moz.build	Mon Oct 21 12:54:56 2024 +0000
@@ -2,12 +2,13 @@
 # License, v. 2.0. If a copy of the MPL was not distributed with this
 # file, You can obtain one at http://mozilla.org/MPL/2.0/.
 
+SOURCES += ["TestFmt.cpp"]
+
 if CONFIG["OS_ARCH"] == "WINNT":
     SOURCES += [
         "TestNativeNtGTest.cpp",
     ]
 
-
 if not CONFIG["MOZ_TSAN"]:
     SOURCES += [
         "TestStackWalk.cpp",
